app/src/db/DatabaseCon.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Src\db;
+
+use PDO;
+
+class DatabaseCon
+{
+    private $dbConnection = null;
+    private static $instance = null;
+
+    public function __construct()
+    {
+        $host = getenv('DB_HOST');
+        $port = getenv('DB_PORT');
+        $db = getenv('DB_DATABASE');
+        $user = getenv('DB_USERNAME');
+        $password = getenv('DB_PASSWORD');
+
+        try {
+            $this->dbConnection = new PDO(
+                "mysql:host=$host;port=$port;charset=utf8mb4;dbname=$db",
+                $user,
+                $password
+            );
+        } catch (\PDOException $e) {
+            return $e->getMessage();
+        }
+    }
+
+    public static function getInstance()
```

Comment:
@jjsimonAB all methods have to have their doc and retury type thats the fun part